### PR TITLE
Support auto title stories

### PR DESCRIPTION
**Pull request by yannbf**
Issue: #19 

<details>
  <summary>📦 Published PR as canary version: <code>0.0.2-canary.35.8a26a58.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @storybook/test-runner@0.0.2-canary.35.8a26a58.0
  # or 
  yarn add @storybook/test-runner@0.0.2-canary.35.8a26a58.0
  ```
</details>

### CODE REVIEW

File: src/playwright/transformPlaywright.ts [hunk 1]
```diff
@@ -19,12 +23,43 @@ export const testPrefixer = template(
...
+  const normalizedStoriesEntries = normalizeStories(main.stories, {
+    configDir,
+    workingDir,
+  }).map((specifier) => ({
+    ...specifier,
+    importPathMatcher: new RegExp(specifier.importPathMatcher),
+  }));
+
+  // filename: /Users/xxxx/test-runner/stories/basic/Button.stories.js
+  const filePath = `.${filename.split(workingDir)[1]}`;
```
Comment (yannbf) [inline src/playwright/transformPlaywright.ts:48]: @shilman I might need a hand thinking a proper way to get the file path

Comment (yannbf) [reply src/playwright/transformPlaywright.ts]: I did not, thanks for the great tip! :chefskiss:

**Review by shilman — APPROVED** [review]
💯 looking great @yannbf

**Comment by shilman** [comment]
Except failing tests 🙈 😅

**Comment by yannbf** [comment]
> Except failing tests 🙈 😅

That's on purpose, I wanted to discuss with you if the approach was the final one, then I'd write the test

**Pull request merged**